### PR TITLE
Migrating recordings to the assets repo

### PR DESCRIPTION
**Pull request by kinelski**
In order to reduce the quickly growing size of our repository, the Engineering System team requested us to move our test recordings to the [Azure SDK Assets](https://github.com/Azure/azure-sdk-assets) repo.

For now, I'm only moving data plane libraries owned by the Azure SDK for .NET team. Further instructions on how this changes the workflow have been sent out via email to our team members but they're also documented in the [Test Framework README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/README.md#recording).

### CODE REVIEW

File: sdk/appconfiguration/Azure.Data.AppConfiguration/tests/SessionRecords/ConfigurationLiveTests/AddExistingSetting.json [hunk 1]
```diff
@@ -1,153 +0,0 @@
-{
-  "Entries": [
-    {
-      "RequestUri": "https://shrejaappconfiguration-azconfig-net.azconfig.io/kv/key-1022856440?label=test_label\u0026api-version=1.0",
-      "RequestMethod": "PUT",
-      "RequestHeaders": {
```
Comment (ShivangiReja) [inline sdk/appconfiguration/Azure.Data.AppConfiguration/tests/SessionRecords/ConfigurationLiveTests/AddExistingSetting.json:6]: I notice that the test recordings are removed from this repo, and I'm curious why there is no pull request or inclusion of these test recordings in the [Azure SDK Assets repo](https://github.com/Azure/azure-sdk-assets)?

Comment (kinelski) [reply sdk/appconfiguration/Azure.Data.AppConfiguration/tests/SessionRecords/ConfigurationLiveTests/AddExistingSetting.json:6]: Recordings are pushed directly without the need for approval, so there's no pull request. Also, they're not pushed to a branch. Instead, the commit they are associated with is referenced via a [git tag](https://docs.github.com/en/rest/git/tags?apiVersion=2022-11-28).

Here's an example: [AppConfig tag](https://github.com/Azure/azure-sdk-assets/releases/tag/net%2Fappconfiguration%2FAzure.Data.AppConfiguration_2f82b09ab4) and its [associated commit](https://github.com/Azure/azure-sdk-assets/commit/2f82b09ab46bb8be00b544ac1f3ada211f3f577e).

Comment (ShivangiReja) [reply sdk/appconfiguration/Azure.Data.AppConfiguration/tests/SessionRecords/ConfigurationLiveTests/AddExistingSetting.json:6]: Ahhh, thanks for the clarification!

**Review by heaths — APPROVED** [review]
How are you finding the right recordings? I use recorded tests for KV and none of those files are in here.

**Comment by kinelski** [comment]
> How are you finding the right recordings? I use recorded tests for KV and none of those files are in here.

Discussed offline. Key Vault recordings are included in the PR, but the UI seems to be struggling with so many files. The changes are split into 11 commits, one per service directory, so I suggest filtering by commit for anyone who's experiencing issues with the interface.

**Pull request merged**